snapshots/overlay/overlay.go

```diff
@@ -518,6 +534,13 @@ func (o *snapshotter) mounts(s storage.Snapshot) []mount.Mount {
 		options = append(options, "userxattr")
 	}
 
+	switch o.metacopy {
+	case metacopyEnabled:
+		options = append(options, "metacopy=on")
+	case metacopyDisabled:
+		options = append(options, "metacopy=off")
+	}
+
```

Comment:
`userxattr` and `metacopy=on` cannot be specified together
https://github.com/torvalds/linux/blob/4b0986a3613c92f4ec1bdc7f60ec66fea135991f/fs/overlayfs/super.c#L734

Reply:
Yes, thanks, I'll add this check